proxy/interceptor.go

```diff
@@ -619,6 +620,58 @@ func visitPayloads(
 				return err
 			}
 
+		case *errordetails.MultiOperationExecutionFailure:
+
```

Comment:
So a bunch of stuff got added to the failure interceptor, which previously I guess did not start from the top of anything that contains a failure, like the payload one does. Is that intentional? Assuming it is, is starting from the top like this a bad thing?